src/plugins/home/public/application/components/home_app.js

```diff
@@ -75,7 +75,7 @@ export function HomeApp({ directories, solutions }) {
               localStorage={localStorage}
               urlBasePath={getBasePath()}
               telemetry={telemetry}
-              http={http}
+              indexPatternService={indexPatternService}
```

Comment:
optional: maybe we should provide a single method instead of passing the whole service?

Reply:
good point! changed!